content/en/tracing/trace_collection/trace_context_propagation/dotnet.md

```diff
@@ -34,8 +34,9 @@ The environment variable values are comma-separated lists of header styles enabl
 
 **Notes**: 
 
-- Starting from version [2.22.0](https://github.com/DataDog/dd-trace-dotnet/releases/tag/v2.22.0), the default injection style is `tracecontext, Datadog`, so the W3C Trace Context is used, followed by the Datadog headers. Prior to version 2.22.0, only the `Datadog` injection style is enabled.
+- Starting from version [2.48.0](https://github.com/DataDog/dd-trace-dotnet/releases/tag/v2.48.0), the default propagation style is `Datadog, tracecontext`, so the Datadog headers are used, followed by the W3C Trace Context. Prior to version 2.48.0, the order was `tracecontext, Datadog` for both extraction and injection propagation.
 - Starting from version [2.42.0](https://github.com/DataDog/dd-trace-dotnet/releases/tag/v2.42.0), when multiple extractors are specified, the `DD_TRACE_PROPAGATION_EXTRACT_FIRST=true` configuration specifies whether context extraction should exit immediately upon detecting the first valid `tracecontext`. The default value is `false`.
+- Starting from version [2.22.0](https://github.com/DataDog/dd-trace-dotnet/releases/tag/v2.22.0), the default injection style is `tracecontext, Datadog`, so the W3C Trace Context is used, followed by the Datadog headers. Prior to version 2.22.0, only the `Datadog` injection style is enabled.
```

Comment:
Maybe it would be better if we included the 2.22.0 part in there too rather than a separate bullet, to try to reduce confusion?

```suggestion
- Starting from version [2.48.0](https://github.com/DataDog/dd-trace-dotnet/releases/tag/v2.48.0), the default propagation style is `Datadog, tracecontext`, so the Datadog headers are used, followed by the W3C Trace Context. Prior to version 2.48.0, the order was `tracecontext, Datadog` for both extraction and injection propagation.  Prior to version [2.22.0](https://github.com/DataDog/dd-trace-dotnet/releases/tag/v2.22.0), only the `Datadog` injection style was enabled.
- Starting from version [2.42.0](https://github.com/DataDog/dd-trace-dotnet/releases/tag/v2.42.0), when multiple extractors are specified, the `DD_TRACE_PROPAGATION_EXTRACT_FIRST=true` configuration specifies whether context extraction should exit immediately upon detecting the first valid `tracecontext`. The default value is `false`.
```

Reply:
+1